packages/authentication/test/acceptance/basic-auth.ts

```diff
@@ -0,0 +1,99 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  Application,
+  Server,
+  api,
+  OpenApiSpec,
+  ParameterObject,
+  ServerRequest,
+  ServerResponse,
+  parseOperationArgs,
+  writeResultToResponse,
+  ParsedRequest,
+  OperationArgs,
+  FindRoute,
+  InvokeMethod,
+  UserProfile,
+} from '@loopback/core';
+import {expect, Client, createClientForServer} from '@loopback/testlab';
+import {givenOpenApiSpec} from '@loopback/openapi-spec-builder';
+import {inject, Constructor, Context} from '@loopback/context';
+import {authenticate} from '../..';
+const strategy = require('passport-http').BasicStrategy;
+
+describe('Authentication with passport-http', () => {
+  let app: Application;
+  beforeEach(givenAppWithController);
+
+  it ('authenticates', () => {
+
+  });
+
+  function givenAppWithController() {
+    givenAnApplication();
+    app.bind('application.name').to('SequenceApp');
+
+    const apispec = givenOpenApiSpec()
+      .withOperation('get', '/who-am-i', {
+        'x-operation-name': 'whoAmI',
+        responses: {
+          '200': {
+            type: 'string',
+          },
+        },
+      })
+      .build();
+
+    @api(apispec)
+    class MyController {
+      constructor(@inject('authentication.user') private user: UserProfile) {}
+
+      @authenticate('BasicStartegy')
```

Comment:
Typo? `BasicStartegy` 

This may be out of scope of this pull request, but shouldn't applications configure Basic HTTP authentication via the Swagger/OpenAPI spec?

@raymondfeng @ritch Thoughts?